src/libs/ajax.js

```diff
@@ -51,11 +52,20 @@ const withCancellation = wrappedFetch => async (...args) => {
   }
 }
 
+const captureRetryFailure = (...args) => {
+  const errorAddress = _.flow(
+    _.values,
+    _.find(v => _.includes(v, args[0]) && v)
+  )([..._.values(getConfig()), 'https://storage.googleapis.com/'])
+  Ajax().Metrics.captureEvent(Events.requestFailed, { test: errorAddress })
```

Comment:
A more descriptive name for this value would be useful when looking at data in Mixpanel.

```suggestion
  Ajax().Metrics.captureEvent(Events.requestFailed, { requestRoot: errorAddress })
```